wgengine/router/callback.go

```diff
@@ -41,6 +50,10 @@ func (r *CallbackRouter) Set(rcfg *Config) error {
 	if r.rcfg.Equal(rcfg) {
 		return nil
 	}
+	if r.InitialMTU != 0 {
+		rcfg.NewMTU = int(r.InitialMTU)
+		r.InitialMTU = 0
+	}
```

Comment:
`r.InitialMTU` is mutated but not guarded by `r.mu` here. And `Set` can be called concurrently, so we do need to consider this.

Also now that I think about it, probably worth mentioning in the comment on the `InitialMTU` field that the caller should never mutate it after creating the `CallbackRouter` struct.

Reply:
I guess the other option (up to you, no preference here) is to have `r.InitialMTU` be configuration that's only set at creation and never modified, and add a new `r.didSetMTU` `bool` that is protected by the mutex and you use to avoid setting twice.  That way it's ~impossible for a caller to accidentally cause a race condition.

Reply:
Huh, I think line 48 is r.mu.Lock() so r.InitialMTU is mutated only under the lock. But the flag version is better anyway. I didn't like exporting a thing protected by an unexported lock.